src/main/react/src/containers/TheHeaderDropdownNotif.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { CBadge, CDropdown, CDropdownItem, CDropdownMenu, CDropdownToggle, CProgress } from "@coreui/react";
+import CIcon from "@coreui/icons-react";
+
+const TheHeaderDropdownNotif = () => {
+  const itemsCount = 5;
+  return (
+    <CDropdown inNav className="c-header-nav-item mx-2">
+      <CDropdownToggle className="c-header-nav-link" caret={false}>
+        <CIcon name="cil-bell" />
+        <CBadge shape="pill" color="danger">
+          {itemsCount}
+        </CBadge>
+      </CDropdownToggle>
+      <CDropdownMenu placement="bottom-end" className="pt-0">
+        <CDropdownItem header tag="div" className="text-center" color="light">
+          <strong>You have {itemsCount} notifications</strong>
+        </CDropdownItem>
+        <CDropdownItem>
+          <CIcon name="cil-user-follow" className="mr-2 text-success" /> New user registered
```

Comment:
see if you can remove stuff we'll never use. It's making the code base messy.

Reply:
Just removed the extraneous stuff from the header, that should cover most of it.